openslides/users/apps.py

```diff
@@ -32,6 +35,10 @@ def ready(self):
             get_permission_change_data,
             dispatch_uid='users_get_permission_change_data')
 
+        # Disconnect the last_login signal
+        if 'DISABLE_LAST_LOGIN_FIELD' in dir(settings) and settings.DISABLE_LAST_LOGIN_FIELD:
```

Comment:
Please use settings variable as shown [here](https://docs.djangoproject.com/en/1.11/topics/settings/#using-settings-in-python-code).

Reply:
Done.